libc/printf/test.yaml

```
@@ -0,0 +1,14 @@
+test:
+  type: unity
+  tests:
+    - name: unit
```

We always want to leave a comment, why do we use an unusual approach, like this one:
```suggestion
    # Executing test-printf on 'armv7m4-stm32l4x6-nucleo' takes ~9 min, that's why we want to run it only during nightly campaigns 
    - name: unit
```